### ISSUE TITLE
Can't download book

### ISSUE THREAD

**Issue by dnstockdale** [rmcrackan/Libation#343]
**Describe the bug**
I'm not sure this is a bug, but it's an issue I don't know how to fix, so...

I was just told about Libation a couple days ago, so yesterday I started downloading and decrypting all my Audible books, I think somewhere around 40 or so. I got about half downloaded before I got tired and decided to go to bed; I meant to leave my computer on and let them download overnight, but I accidentally put my computer to sleep while Libation was in the middle of downloading a book. 

When I booted up this morning, I closed the program, reopened it, cleared the download cue to start fresh, then clicked on the book it had been in the middle of downloading. I got the pop-up message that an error occurred while the book was processing, along with the options of abort, retry, and ignore. I didn't want to abort or ignore, so I selected retry and went on to download the rest of the 40 books that hadn't downloaded yet. 

When all books but the one were downloaded and decrypted, I thought the program would automatically retry the error one, but it didn't. After a couple hours I cleared the cue, closed the program, reopened it, and clicked on the yellow stoplight next to the error book. I got the same pop-up error message, so I selected retry and waited. The day passed, and it never retried to process. So here I am posting about the issue because I don't know how to get it to download.

**Platform**
Windows 10 on an HP Workstation laptop. Libation Classic 8.3.6.1

Let me know if I can provide other information. Hopefully there's an easy fix.

**Comment by Mbucari** [comment]
I'm sorry this is happening to you. Retry means that libation will attempt to liberate that book again when you _initiate_ a liberate. It won't attempt it on its own sometime later.

Libation saves downloads in progress in case the connection is interrupted while downloading. It sounds like your in progress file got corrupted. Delete all files inside `%Temp%\Libation\DownloadsInProgress`

**Comment by dnstockdale** [comment]
Success! 

I found the location you specified and tried to delete the files inside but one file for the error book refused to be deleted. It said it couldn't be removed because it was open in Libation--and it actually was at the time (whoops), so I cleared the download cue and closed the program. Then I tried to delete that last file again, but I got the same pop-up message: it couldn't be removed because it was open in Libation. I brought up Task Manager and looked to see if Libation was still running in the background or something, and it was indeed listed, so I end-tasked it and tried again to remove the file--and it worked! I opened Libation and successfully downloaded the book.

Thanks for the help!!
